Failing tests on CI due to wrong Qt version

(Documenting an issue I encountered and about to put in a fix)

During testing, CircleCI activates the virtual environment for the launcher (note this is not the root project venv) as follows:
```
cd launcher/
make venv && source .venv/bin/activate
make check
```
This means that it's using the PyQt5 venv Makefile target during testing. However, the SDW_UPDATER_QT environment variable isn't set, meaning that the version of QT defaults to the version found by `launcher/sdw_util/Util.get_qt_version()`, which, [when no environment variable is set](https://github.com/freedomofpress/securedrop-workstation/blob/d1e8c98cc099152c7e8957c97248258a5dec6df7/launcher/sdw_util/Util.py#L153-L164), is `4`.  

Predicatable CI sadness when the module attempts to import the wrong version of PyQt: 

```
sdw_updater_gui/UpdaterApp.py:13: in <module>
    from PyQt4.QtGui import QDialog
E   ModuleNotFoundError: No module named 'PyQt4'
```

CI sadness 2 (same job):
```
./launcher/tests/test_updaterapp.py:19:5: F401 'PyQt5.QtWidgets.QDialog' imported but unused
./launcher/tests/test_updaterapp.py:20:5: F401 'PyQt5.QtCore.QThread' imported but unused
./launcher/tests/test_updaterapp.py:20:5: F401 'PyQt5.QtCore.pyqtSignal' imported but unused
./launcher/tests/test_updaterapp.py:20:5: F401 'PyQt5.QtCore.pyqtSlot' imported but unused
./launcher/tests/test_updaterapp.py:21:5: F401 'sdw_updater_gui.UpdaterAppUiQt5.Ui_UpdaterDialog' imported but unused
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -15,6 +15,8 @@ jobs:
             sudo apt install file
             make flake8
             make black
+          environment:
+            SDW_UPDATER_QT: 5
   buildrpm:
     docker:
       - image: circleci/python:3.7-buster

